doc/api/api.raml

```diff
@@ -0,0 +1,31 @@
+#%RAML 1.0
+---
+title: Metamaps
+version: v2
+baseUri: http://metamaps.cc/api/v2
+mediaType: application/json
+
+securitySchemes:
+  - oauth_2_0:
+      description: |
+        OAuth 2.0 implementation
+      type: OAuth 2.0
+      settings:
+        authorizationUri: https://metamaps.cc/api/v2/oauth/authorize
+        accessTokenUri: https://metamaps.cc/api/v2/oauth/token
+        authorizationGrants: [ authorization_code, password, client_credentials, implicit, refresh_token ]
```

Comment:
@connor without diving too deep into raml, does this look right for out implementation? are we using OAuth2 or OAuth1? do you know what values authorizationGrants should have?

I basically just copy pasted this from http://raml.org/developers/whats-new-raml-10